### PR TITLE
AAE-1499 apply variables changes for m6

### PR DESCRIPTION
**Pull request by ffazzini**

### CODE REVIEW

File: example-dmn-runtime-service/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,3 @@
-FROM quay.io/alfresco/alfresco-dmn-runtime-service:7.1.0.M5
+FROM quay.io/alfresco/alfresco-dmn-runtime-service:develop
```
Comment (miguelruizdev) [inline example-dmn-runtime-service/Dockerfile:1]: Are we changing this to 7.1.0.M6 after the release?

**Review by miguelruizdev — APPROVED** [review]
LGTM

File: example-dmn-runtime-service/Dockerfile (hunk 1 above)
Comment (ffazzini) [reply example-dmn-runtime-service/Dockerfile:1]: yep

Comment (ffazzini) [reply example-dmn-runtime-service/Dockerfile:1]: or just a tag

**Pull request merged**